src/components/FlawForm.vue

```diff
@@ -290,21 +290,21 @@ const onReset = () => {
           <button
             type="button"
             class="btn btn-secondary"
-            @click="showSummary = !showSummary"
+            @click="showSummary = !showSummary; flaw.summary = '';"
           >
             {{ showSummary ? 'Remove Description' : 'Add Description' }}
           </button>
           <button
             type="button"
             class="btn btn-secondary"
-            @click="showStatement = !showStatement"
+            @click="showStatement = !showStatement; flaw.statement = '';"
           >
             {{ showStatement ? 'Remove Statement' : 'Add Statement' }}
           </button>
           <button
             type="button"
             class="btn btn-secondary"
-            @click="showMitigation = !showMitigation"
+            @click="showMitigation = !showMitigation; flaw.mitigation = '';"
           >
```

Comment:
May be better to make these statements into toggle functions and to only reset if they are hidden, as of now these are hardcoded to the right functionality only because of the initial values